eclipse-theia/theia
# [SCM] Fix SCM status bar commands

#### What it does

Pass command arguments to SCM status bar commands. Pass `Command.id` to SCM status bar commands if `Command.command` is undefined.

#### How to test

1. Clone the test plugin: https://github.com/vinokurig/Test.git
2. Compile the plugin and start it as a hosted instance.
3. Select the `/` SCM repository (this is a view of a dummy SCM repository):
![screenshot-localhost-3030-2019 09 06-17-34-24](https://user-images.githubusercontent.com/7668752/64436695-892fe000-d0cd-11e9-8f13-ccd7492bcbb7.png)
4. Click `title` button in the status bar:
![screenshot-localhost-3030-2019 09 18-15-18-52](https://user-images.githubusercontent.com/7668752/65147946-d7c36f80-da27-11e9-81bd-70dcc5f1b4c9.png)

See the notification

fixes https://github.com/eclipse/che/issues/13974
#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
